README.md

```diff
@@ -26,15 +26,59 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
 
 The Iceberg documentation site is actually constructed from two hugo sites. The first, is the landing page. The second site, 
 is the documentation site which contains the full Iceberg documentation, including the javadoc. The landing page and
-documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively.
+documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively. The Javadocs are in the `./javadoc` directory.
 
-# Landing Page Deployment
+# How to Contribute
+
+## Submitting Pull Requests
+
+All pull requests to update the latest documentations should be submitted to the [Iceberg repository](https://github.com/apache/iceberg).
+When Iceberg publishes a new release, the release manager copies all the documentation to this repository and cut a new version branch.
+
+Changes to the landing page should be submitted to this repository against the `main` branch.
+
+Changes to the documentation of old Iceberg versions should be submitted to this repository against the specific version branch.
+
+## Reporting Issues
+
+All issues related to documentation and website should still be submitted to the [Iceberg repository](https://github.com/apache/iceberg).
+
+## Running Locally
+
+To start the landing page site locally, clone this repository and run the following.
+```shell
+git clone git@github.com:apache/iceberg-docs.git
+cd landing-page && hugo serve
+```
+
+To start the documentation site locally, clone this repository and run the following.
+```shell
+git clone git@github.com:apache/iceberg-docs.git
+git submodule update --init
+cd docs && hugo serve
+```
+
+## Viewing Latest Website
+
+If you would like to see how the latest website looks based on the documentation in the Iceberg repository, you can copy docs to this repository by:
```

Comment:
need to refine this workflow a bit, does the `docs/content/docs` path contains all the documentations user might modify?

Reply:
Looks like there is another part in the `landing-page/common`

Reply:
okay so I assume the iceberg repo would have a `documentation` folder and `common` and `docs` folder inside mapping to the 2 sites here.

Reply:
Yep, exactly right. In `apache/iceberg` we would have two directories that are intuitively named like `documentation/common` and `documentation/versioned` and they would hold the contents in [docs/content/docs](https://github.com/apache/iceberg-docs/tree/main/docs/content/docs) and [landing-page/content/common](https://github.com/apache/iceberg-docs/tree/main/landing-page/content/common), respectively. The release manager would then just do a full hard-copy over of those two directories (might be worth mentioning that it should be a full copy, meaning deleted files are removed in the destinations as well).

Reply:
Once this PR and PR #27 is merged, I'll go ahead and open the PR against `apache/iceberg` to add those `documentation/common` and `documentation/versioned` directories.

Reply:
got it, let me update based on this info